import/popups/qmldir

```diff
@@ -24,3 +24,5 @@ PromptLabel 1.0 PromptLabel.qml
 DownloadMenuItem 1.0 DownloadMenuItem.qml
 ShareLinkPage 1.0 ShareLinkPage.qml
 ShareTextPage 1.0 ShareTextPage.qml
+WebrtcPermissionInterface 1.0 WebrtcPermissionInterface.qml
```

Comment:
nitpick: I'd prefer to group the Interface types together, and the Dialog types together in the qmldir, just for readability

Reply:
i.e. move line 27 to 12/31, and move line 28 to 20/21.

Reply:
Done. Thanks.